webpack.config.js

```diff
@@ -53,9 +55,12 @@ module.exports = (env = {}) => {
     context: bundleEntryDir,
     entry: {
       // Use arrays for every entry to allow for hot reloading.
-      base: ['@babel/polyfill', './base.js'],
-      channel_edit: ['./channel_edit.js'],
-      channel_list: ['./channel_list.js'],
+      channel_edit: [
+        path.resolve(djangoProjectDir, 'contentcuration', 'assets/channelEdit/index.js'),
```

Comment:
Makes pretty much no difference, but couldn't you have used the `assetsDir` variable you created above?